tools/fix_auth/fix_auth.rb

```diff
@@ -67,6 +67,9 @@ def fix_database_passwords
         ActiveRecord::Base.logger = Logger.new("#{options[:root]}/log/fix_auth.log")
         ActiveRecord::Base.establish_connection(db_attributes(database))
       end
+
+      puts "database: #{options[:database]}"
```

Comment:
Does this get lost when the remaining messages are written to the screen from the different models?

Can you post a sample of what it looks like now?

Thanks!

Reply:
I did change all of these to upper case, but it was harder for me to read for some reason.

I'll add processing, but I'm thinking lowercase for now.
Will also update the usage up top.